README.adoc

```diff
@@ -205,25 +205,25 @@ include::finish/backendServices/src/main/java/io/openliberty/guides/event/dao/Ev
 ----
 
 To use the entity manager at runtime, inject it into our CDI bean through the
-[hotspot=14 file=0]`@PersistenceContext` annotation. The entity manager interacts with the persistence context. 
+[hotspot=PersistenceContext file=0]`@PersistenceContext` annotation. The entity manager interacts with the persistence context. 
 Every `EntityManager` instance is associated with a persistence context. The persistence context 
 manages a set of entities and is aware of the different states that an entity can have.
 The persistence context synchronizes with the database when a transaction commits.
 
-The [hotspot=12-43 file=0]`EventDao` class has a method for each CRUD operation, so let's break them down:
+The [hotspot=EventDao file=0]`EventDao` class has a method for each CRUD operation, so let's break them down:
 
-* The [hotspot=17-19 file=0]`createEvent()` method persists an instance of the [hotspot=17-70 file=1]`Event` entity class to the data store by
-calling the [hotspot=18 file=0]`persist()` method on an `EntityManager` instance. The entity instance becomes managed
+* The [hotspot=createEvent file=0]`createEvent()` method persists an instance of the [hotspot=Event file=1]`Event` entity class to the data store by
+calling the [hotspot=Persist file=0]`persist()` method on an `EntityManager` instance. The entity instance becomes managed
 and changes to it will be tracked by the entity manager.
 
-* The [hotspot=21-23 file=0]`readEvent()` method returns an instance of the [hotspot=17-70 file=1]`Event` entity class with the specified primary 
-key by calling the [hotspot=22 file=0]`find()` method on an `EntityManager` instance. If the event instance is found, it
+* The [hotspot=readEvent file=0]`readEvent()` method returns an instance of the [hotspot=Event file=1]`Event` entity class with the specified primary 
+key by calling the [hotspot=Find file=0]`find()` method on an `EntityManager` instance. If the event instance is found, it
 is returned in a managed state, but, if the event instance is not found, `null` is returned.
 
-* The [hotspot=33-35 file=0]`readAllEvents()` method demonstrates an alternative way to retrieve event objects 
+* The [hotspot=readAllEvents file=0]`readAllEvents()` method demonstrates an alternative way to retrieve event objects 
 from the database. This method returns a list of instances of the `Event` entity class by 
```

Comment:
The `Event` tag references the same entity as the previous two hotspots, but this tag is not a hotspot. This is a consistency issue.